src/index.tsx

```diff
@@ -92,60 +98,18 @@ const Main = () => {
 
     const message = messagesData.find((msg) => msg.id === messageId);
     if (message) {
-      if (!message.bookmark) {
-        toast(`Přidáno do Záložky.`, {
-          position: 'bottom-center',
-          autoClose: 2000,
-          hideProgressBar: true,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          progress: 0,
-          theme: 'light',
-          transition: Slide,
-          closeButton: false,
-          style: {
-            width: '220px',
-            height: '20px',
-            background: '#1da1f2',
-            color: '#ffffff',
-            fontSize: '18px',
-            textAlign: 'center',
-          },
-        });
-      } else {
-        toast(`Odebráno ze Záložky.`, {
-          position: 'bottom-center',
-          autoClose: 2000,
-          hideProgressBar: true,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          progress: 0,
-          theme: 'light',
-          transition: Slide,
-          closeButton: false,
-          style: {
-            width: '220px',
-            height: '20px',
-            background: '#1da1f2',
-            color: '#ffffff',
-            fontSize: '18px',
-            textAlign: 'center',
-          },
-        });
-      }
+      message.bookmark
+        ? showToast('Odebráno ze Záložky.')
```

Comment:
Krásně se to zkrátilo přesunutím toastu, že? 🫶

Reply:
ráda jsem si na konkrétním příkladu vyzkoušela napsat vlastní `hook` 😀